research/object_detection/g3doc/tf2.md

```diff
@@ -82,3 +82,8 @@ We provide a large collection of models that are trained on COCO 2017 in the
       TPU compatible detection pipelines</a><br>
 *   <a href='tf2_training_and_evaluation.md'>
       Training and evaluation guide (CPU, GPU, or TPU)</a><br>
+
+## Community Contributed Guides
+*   [Fine-tuning a SSD_MobileNetV2 model with Cloud TPUs from Colab](https://github.com/sayakpaul/E2E-Object-Detection-in-TFLite/blob/master/Training_a_pets_detector_model_within_minutes_with_TFOD_API.ipynb) (non-eager mode)
+*   [Converting a detection model to TFLite and running inference](https://github.com/sayakpaul/E2E-Object-Detection-in-TFLite/blob/master/Object_Detection_in_TFLite.ipynb)
+*   [Preparing inputs, fine-tuning a custom object detection model in Colab, and converting to TFLite](https://github.com/sayakpaul/E2E-Object-Detection-in-TFLite/tree/master/colab_training) (non-eager mode)
```

Comment:
These notebooks are written with TF1 ODT API so I'd suggest rewriting them in TF2 ODT APU with a different model architecture that's already supported in TF2 (e.g. Mobilenetv2-SSD)

Reply:
@khanhlvg notebooks under `colab_training` are based on TensorFlow Object Detection API (TF 1) while all the other notebooks are based on TensorFlow Object Detection API (TF 2) based on the MobileNetV2 architecture.

Reply:
As you're adding to the `tf2.md` file, the notebook linked from this page should be using TF2 only to avoid confusion. I'd suggest moving the third link to `tf1.md`

Reply:
Fair enough. I will do that.

Reply:
Taken care of.